app/controllers/answers_controller.rb

```diff
@@ -1,6 +1,6 @@
 class AnswersController < ApplicationController
   def create
-    SetAnswer.run! answer_params
+    SetAnswer.run! answer_params.to_unsafe_h
```

Comment:
Waarom `to_unsafe_h`? Benodigde params worden al gepermit, en als ik [deze blogpost](http://eileencodes.com/posts/actioncontroller-parameters-now-returns-an-object-instead-of-a-hash/) lees, is `to_unsafe_h` alleen nodig als je niet expliciet gepermitte params wilt meegeven.